mono/metadata/custom-attrs.c

```diff
@@ -1316,6 +1316,7 @@ mono_reflection_create_custom_attr_data_args_noalloc (MonoImage *image, MonoMeth
 fail:
 	mono_error_set_generic_error (error, "System.Reflection", "CustomAttributeFormatException", "Binary format of the specified custom attribute was invalid.");
 	g_free (arginfo);
+	g_free (named_args);
```

Comment:
I don't see where we're allocating `named_args` in this function? isn't it the responsibility of the caller?

Reply:
Presumably it is supposed to be `namedargs` without the underscore.
https://github.com/mono/mono/pull/13855/files#diff-7781be09cd2ddfae798d13ecb23a63c6R1224

I suggest, um, renaming the args. `out_namedargs` and `namedargs`?

Reply:
And `namedargs` will need null initialization also.

Reply:
Or put a star on it and ignore the rest I said, `g_free(*named_args);` that should do, but double check.

Reply:
No the last point is bad. We don't write the out parameter until the very end of success. So the first part -- null initialize the local and add underscore to the free.